Formula/groovy@2.rb

```diff
@@ -0,0 +1,32 @@
+class GroovyAT2 < Formula
+  desc "Java-based scripting language"
+  homepage "https://www.groovy-lang.org/"
+  url "https://dl.bintray.com/groovy/maven/apache-groovy-binary-2.5.9.zip"
+  sha256 "fea7dc321a3029c47ffa4aa165055d2bcc78bc280fac4e70ac131c717e45b89b"
+
+  bottle :unneeded
+
+  keg_only :versioned_formula
+
+  # Groovy 2.5 requires JDK8+ to build and JDK7 is the minimum version of the JRE that we support.
+  depends_on :java => "1.7+"
```

Comment:
@raffecat I think this line needs to be:
```suggestion
  depends_on "openjdk"
```
due to recent migration efforts. Hopefully someone will confirm